packages/dashboard/src/app/views/exploreTemplates/content/templateGridItem.js

```diff
@@ -113,20 +115,22 @@ const TemplateGridItem = forwardRef(
                   {TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS[status]}
                 </Button>
 
-                <Button
-                  size={BUTTON_SIZES.SMALL}
-                  type={BUTTON_TYPES.PRIMARY}
-                  onClick={onCreateStory}
-                  disabled={!onCreateStory}
-                  tabIndex={tabIndex}
-                  ariaLabel={sprintf(
-                    /* translators: %s: template title.*/
-                    __('Create new story from %s', 'web-stories'),
-                    title
-                  )}
-                >
-                  {__('Use template', 'web-stories')}
-                </Button>
+                {apiCallbacks?.createStoryFromTemplate && (
+                  <Button
+                    size={BUTTON_SIZES.SMALL}
+                    type={BUTTON_TYPES.PRIMARY}
+                    onClick={onCreateStory}
+                    disabled={!onCreateStory}
```

Comment:
While we're at it: This seems a bit pointless 🤔 

Right now, `onCreateStory` is never false.

And with the proposed change above we don't render that component in the first place when it's not provided.

So we could just remove this.